### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by markmur**

### CODE REVIEW

File: helpers/lowercase.js [hunk 1]
```diff
@@ -3,9 +3,12 @@
  * @method lowercase
  * @param  {String} str
  * @return {String} Returns string in lowercase
+ * @alias toLowerCase
  * @example {{lowercase 'TEXT'}} // Outputs 'text'
  */
 module.exports.lowercase = function lowercase(str) {
   if (!str || typeof str !== 'stirng') return '';
```
Comment (danieldmo) [inline helpers/lowercase.js]: string check  is wrong there

Comment (markmur) [inline helpers/lowercase.js]: oops

File: tests/helpers/toLowerCase.test.js [hunk 2]
```diff
@@ -1,11 +0,0 @@
-var helpers = require('../../js/main.js');
```
Comment (danieldmo) [inline tests/helpers/toLowerCase.test.js]: did the test get removed?

File: helpers/lowercase.js (hunk 1 above)
Comment (markmur) [inline helpers/lowercase.js]: Fixed on master

File: tests/helpers/toLowerCase.test.js (hunk 2 above)
Comment (markmur) [inline tests/helpers/toLowerCase.test.js]: Yeah sorry I removed the helpers folder. All of the helpers are in tests/test.js now. I hadn't realised you'd written one. I didn't write the tests file properly until today

Comment (danieldmo) [inline tests/helpers/toLowerCase.test.js]: I think I liked better having the tests in separate files. it was very easy to find them

Comment (markmur) [inline tests/helpers/toLowerCase.test.js]: Yeah me too, I'll get them back to being like that. I just need to find a way to concat them into the tests file. I don't want to have to require handlebars and chai in every single test

Comment (markmur) [inline tests/helpers/toLowerCase.test.js]: There's 20 passing and 25 pending if you're feeling in the mood to write some tests :P ! .. Just run with `npm test`

Comment (danieldmo) [inline tests/helpers/toLowerCase.test.js]: cant you gulp it before running the tests?

Comment (markmur) [inline tests/helpers/toLowerCase.test.js]: Yep, just haven't done it yet

**Pull request merged**